.eslintrc.js

```diff
@@ -9,4 +9,17 @@ module.exports = {
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
   ],
+  rules: {
```

Comment:
Let me know your thoughts on these rules!
`@typescript-eslint/no-use-before-define` was causing it to error when using a function before it's defined. In Haskell, we normally put the most important function at the top of a file, and helper functions beneath it.

`'@typescript-eslint/member-delimiter-style'` was set to a semi-colon by default so for instance, types had to be laid out like:
```ts
type Fake = {
  key1: string;
  key2: string;
  key2: string;
}
```
It's mostly personal preference, but I much prefer the clean look of no delimiter. Thoughts?

Reply:
Yeah same, only semicolons when absolutely necessary 😄 
Rules seem fine 👍